cassandra-commons/src/main/java/com/mesosphere/dcos/cassandra/common/config/CassandraApplicationConfig.java

```diff
@@ -283,8 +283,8 @@
   public static final int DEFAULT_NATIVE_TRANSPORT_MAX_CONCURRENT_CONNECTIONS_PER_IP = -1;
   public static final int DEFAULT_RPC_MIN_THREADS = 16;
   public static final int DEFAULT_RPC_MAX_THREADS = 2048;
-  public static final int DEFAULT_RPC_SEND_BUFF_SIZE_IN_BYTES = 0;
-  public static final int DEFAULT_RPC_RECV_BUFF_SIZE_IN_BYTES = 0;
+  public static final int DEFAULT_RPC_SEND_BUFF_SIZE_IN_BYTES = 16384;
+  public static final int DEFAULT_RPC_RECV_BUFF_SIZE_IN_BYTES = 16384;
```

Comment:
This wont take effect because this line will set it to 0: https://github.com/mesosphere/dcos-cassandra-service/blob/672cc8cf416ed430b9c7b4e4214d9fc1e9da1c17/cassandra-scheduler/src/dist/conf/scheduler.yml#L130

You should change both the values there.